### PR TITLE
Improvements 2

### PR DESCRIPTION
**Pull request by radujipa**
This PR includes the following enhancements:

 - Allowed `cat ./script-name` for challenge 25 progression
   https://github.com/KanoComputing/peldins/issues/2231  

 - Fixed delayed sound playback for sounds inside the story
   https://github.com/KanoComputing/peldins/issues/2190

 - Added more sounds: story sounds, asset sounds, command sounds, new spell sound
   https://github.com/KanoComputing/peldins/issues/2193

 - Fixed blocking `mv` from renaming items in challenge 14
   https://github.com/KanoComputing/peldins/issues/2189

 - Copy colouring and spelling improvements

### CODE REVIEW

**Comment by radujipa** [comment]
@alex5imon could you please review?

File: linux_story/commands_real.py [hunk 1]
```diff
@@ -148,9 +156,27 @@ def shell_command(real_loc, line, command_word=""):
     if stdout:
         if command_word == "cat":
             print stdout
+
         else:
             print stdout.strip()
 
+    # notifying the SoundManager about the command that was run
+    try:
```
Comment (alex5imon) [inline linux_story/commands_real.py]: This perhaps should be done inside `sounds_manager`. So only `sounds_manager` "know about the sounds.

File: linux_story/helper_functions.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def colour_file_dir(path, f):
     return f
 
 
-def play_sound(object_name):
+def play_sound(sound_name, background=True):
```
Comment (alex5imon) [inline linux_story/helper_functions.py]: Perhaps this should go into `sounds_manger` as well?

File: linux_story/object_sound_manager.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
+# object_sound_manager.py
+#
+# Copyright (C) 2014-2016 Kano Computing Ltd.
```
Comment (alex5imon) [inline linux_story/object_sound_manager.py]: `2016` only

**Comment by alex5imon** [comment]
Much better! yiha!

**Pull request merged**